### PR TITLE
feat: add possibility to not load Mollie script globally

### LINKED ISSUE #4: Feature-Request: possibility to include Mollie-script not on all pages
It would be very nice to be able to load the mollie script, which is loaded right now in the server plugin, not on every page.
For us, it only needs to be included on the checkout page where payment methods get selected by the user.

I see the following needs:
- be able to include the script from client side (not rely on the server plugin).
- maybe provide a way to trigger the script loading manually (which would check if the script is already loaded). This could actually be included in the client-plugin directly.

### LINKED ISSUE #6: Doesn't support client side rendered pages
related to #4 

Because the Mollie script is only inlcuded on the server side, the components currently don't work, if you load a page that is configured to be rendered clientside in Nuxt.

We configure the checkout pages to be rendered client side to not have caching issues for example. 
So the components currently only work in this setup, if you start your session on a server rendered page and then go to checkout. If you for any reason reload the checkout page, the components don't work anymore.

### PR DESCRIPTION
**Pull request by niklaswolf**
fixes #4 and fixes #6

# Changes
- added new option `includeScriptGlobally` which is `true` by default. If set to false, the Mollie-scripts are lazily loaded on the first initialization of the Mollie instance. This is done by exposing a loader-function from the client-side mollie plugin, which loads the mollie scripts and returns a promise. The promise resolves, when the script has been loaded.

Should be fully backwards-compatible, as the components itself are unchanged, only the underlying plugin and composable are changed.

# Test
1. Start the dev-server without changes: when opening `localhost:3000` and `localhost:3000/without` the Mollie script should be loaded like before (server-side)
2. Make changes in Nuxt-Config: change `includeScriptGlobally` to `false`
3. Reload both pages: on the index-page, the components should load just fine. On the `/without` page, the script now should only load after the Button is clicked, not before.

### CODE REVIEW

File: playground/pages/index.vue [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+            <pre>2223 0000 1047 9399</pre>
+        </div>
+        <hr />
+        <div v-if="CreditCardError">Error: {{ CreditCardError }}</div>
+        <div v-if="CreditCardToken">Token: {{ CreditCardToken }}</div>
+    </div>
+
+    <h2>Ideal</h2>
+
+    <ShopwareFrontendsIdeal
```
Comment (fjbender) [inline playground/pages/index.vue:42]: The linter complains here

`The template root requires exactly one element`

File: playground/pages/without.vue [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+</script>
+
+<template>
+    <h1>Load Mollie script lazily</h1>
+    <div>On this page the scripts load lazily, if the button is clicked.</div>
+    <div>To test this, the option <i>isIncludedGlobally: false</i> has to be set in nuxt config.</div>
+
+    <p>
+        <button @click="show = !show">{{show ? 'Hide' : 'Show'}} payment components</button>
+        <p v-if="show">
```
Comment (fjbender) [inline playground/pages/without.vue:15]: The linter complains here `The template root requires exactly one element`

File: playground/pages/without.vue [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+                    }
+                "
+                @error="
+                    (message) => {
+                        CreditCardError = `${message} ❌`
+                    }
+                "
+            />
+        </p>
+    </p>
```
Comment (fjbender) [inline playground/pages/without.vue:33]: The linter complains here `Parsing error: x-invalid-end-tag`

**Review by fjbender — COMMENTED** [review]
I might be not understanding what I'm doing, but there were some things that seemed weird to me - see inline comments :)

File: playground/pages/index.vue (hunk 1 above)
Comment (niklaswolf) [reply playground/pages/index.vue:42]: This rule only applies to Vue2, in Vue3 this is possible :)
But there is no harm in wrapping it in a single root, I'll do that to make the linter happy :) 

https://eslint.vuejs.org/rules/no-multiple-template-root#rule-details
> This rule checks whether template contains single root element valid for Vue 2.

File: playground/pages/without.vue (hunk 2 above)
Comment (niklaswolf) [reply playground/pages/without.vue:15]: See other comment :)

File: playground/pages/without.vue (hunk 3 above)
Comment (niklaswolf) [reply playground/pages/without.vue]: fixed :)

File: playground/pages/index.vue (hunk 1 above)
Comment (niklaswolf) [reply playground/pages/index.vue]: fixed :)

Comment (fjbender) [reply playground/pages/index.vue]: Great, thanks! :)

**Review by fjbender — APPROVED** [review]

**Comment by fjbender** [comment]
@benjamin-ott would love your 2 cents on this one :)

**Pull request merged**